crates/next-dev-tests/tests/integration/next/tailwind/basic/issues/lint TP1004 fs.readFileSync(???__star__0__star__, "utf-8") is very dynamic-58d7af.txt

```diff
@@ -17,7 +17,7 @@ PlainIssue {
                             },
                             CellId {
                                 type_id: ValueTypeId {
-                                    id: 31,
+                                    id: 24,
```

Comment:
Oh man, I missed that these are being injected. These are all per-process, which I why I now remember I was using `vc.dbg().await?`.